src/directives/model/checkbox.js

```diff
@@ -5,17 +5,52 @@ module.exports = {
   bind: function () {
     var self = this
     var el = this.el
+    var trueExp = this._checkParam('true-value')
```

Comment:
To be consistent with `exp` for radio input, maybe we should use `true-exp` and `false-exp`?